sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md

```diff
@@ -4,12 +4,20 @@
 
 ### Features Added
 
```

Comment:
Confirmed  the changelog with network service team member offline. They are good with this changelog.

Reply:
Ok, you are good to merge.